s3tests_boto3/functional/test_s3.py

```diff
@@ -13096,3 +13099,44 @@ def test_get_object_torrent():
         status, error_code = _get_status_and_error_code(e.response)
         assert status == 404
         assert error_code == 'NoSuchKey'
+
+
+# test streaming uploads with sizes below and above the 8MB multipart threshold
+upload_sizes = [7 * 1024 * 1024, 9 * 1024 * 1024]
+
+@pytest.mark.checksum
+@pytest.mark.parametrize("size", upload_sizes)
+def test_streaming_upload_checksum_crc32(size):
```

Comment:
the original 7MB size was below the default multipart threshold of 8MB, so wasn't testing multipart uploads. i parameterized these tests for sizes 7MB and 9MB to cover both